client/game/fruit.py

```diff
@@ -0,0 +1,14 @@
+from tkinter.tix import WINDOW
+import pygame 
+from pygame.math import Vector2
+import game.visuals as vs
```

Comment:
lets keep this consistent across the project. Can we switch this to `as visuals`?